examples/kubernetes-releases/kubernetes1.6.json

```diff
@@ -6,7 +6,7 @@
       "orchestratorRelease": "1.6"
     },
     "masterProfile": {
-      "count": 3,
+      "count": 1,
       "dnsPrefix": "",

```

Comment:
AFAIK, there is difference in template generation according to masterCount 1 or not. 
For masterCount == 1, we don't put a internal load balancer in front. While otherwise, we do.
So i guess having count == 3, is to test more generic scenario.